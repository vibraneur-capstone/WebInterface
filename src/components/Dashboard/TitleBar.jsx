import React from 'react';
import { Button } from 'react-bootstrap';

export default class TitleBar extends React.Component {
    constructor (props) {
        super (props)

        this.state = {

        }
    }

    render () {

        let containerStyle = {
            width: this.props.width,
            height: '30px',
            bottom: '1px solid black',
        }

        let buttonStyle = {
            border: '1px solid gray',
            radius: '50%',
            position: 'relative',
            right: '0px',
            backgroundColor: 'white',
            width: '10px',
            height: '10px',
        }

        return (
            <div style={containerStyle}>
                <Button style={buttonStyle}
                    onClick={this.removePanel}
                >x</Button>
                <Button style={buttonStyle}
                    onClick={this.somethingElse}
                >o</Button>
            </div>
        )
    }
}

