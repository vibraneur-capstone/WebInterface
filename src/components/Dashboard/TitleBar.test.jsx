import React from 'react';

import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import TitleBar from './TitleBar.jsx';


test('testing jest', () => {
    expect(0).toBe(0);
})

configure({adapter: new Adapter() });


describe('Testing TitleBar', () => {
    it ('should render correctly', () => {
        const component = shallow(<TitleBar/>);
        expect(component).toMatchSnapshot();
    })
})