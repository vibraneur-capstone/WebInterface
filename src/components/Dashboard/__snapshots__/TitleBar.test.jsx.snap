// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TitleBar should render correctly 1`] = `ShallowWrapper {}`;
